components/twitter/actions/get-user/get-user.mjs

```diff
@@ -0,0 +1,42 @@
+import twitter from "../../twitter.app.mjs";
+
+export default {
+  key: "twitter-get-user",
+  name: "Get User",
+  description: "Return information about the user specified by ID or screen name parameter",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    twitter,
+    userId: {
+      propDefinition: [
+        twitter,
+        "userId",
+      ],
+      optional: true,
+    },
+    screenName: {
+      propDefinition: [
+        twitter,
+        "screenName",
+      ],
+      optional: true,
+    },
+  },
+  async run() {
+    const {
+      userId,
+      screenName,
+    } = this;
+
+    if (!userId && !screenName) {
+      throw new Error("This action requires either User ID or Screen Name. Please enter one or the other above.");
```

Comment:
I think it's best to leave them as optional since they're not required. It's required to have one or the other, but it doesn't matter which one. Using validation and throwing an informative error is the way I've been instructed to handle either/or props in the past. What do you think?

Reply:
Oh, okay, got it!